src/client/test-run/index.js.mustache

```diff
@@ -4,6 +4,10 @@
 
     var origin = location.origin;
 
+    // NOTE: location.origin doesn't exist in IE11 on Windows 10.10240 LTSB
+    if (!origin)
+        origin = location.protocol + '//' + location.hostname + (location.port ? ':' + location.port : '');
```

Comment:
You can simplify this:
https://github.com/DevExpress/testcafe/pull/2979#pullrequestreview-165947681